Configuration/Eras/python/Era_Run2_2017_trackingLowPU_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+
+Run2_2017_trackingLowPU = cms.ModifierChain(Run2_2017, trackingLowPU)
```

Comment:
Here is the cleanup PR #17644. I suggest we let these two go independently, and then I(?) make a further PR fixing `Run2_2_2017_trackingLowPU` and add the workflow for IBs. (or if this PR gets merged first, I can make these subsequent developments in #17644).